app/config.js

```diff
@@ -8,27 +8,36 @@
  */
 define(function (require) {
 	var constants = {
-        NUMBER_TUTORIAL_QUESTIONS: 3,
-        MARGIN_OF_ERROR: 0.2,
-        TUTORIAL_MODE: true,
-        TIME_LIMIT: 600,
-        FEEDBACK_DELAY: 10000,
-        DATABASE_USER: __dirname + "/database/user.sqlite",
-        DATABASE_SLIMSTAMPEN: __dirname + "/database/slimstampen.sqlite"
+		NUMBER_TUTORIAL_QUESTIONS: 3,
+		MARGIN_OF_ERROR: 0.2,
+		TUTORIAL_MODE: true,
+		TIME_LIMIT: 600,
+		FEEDBACK_DELAY_CORRECT: 0.5,
+		FEEDBACK_DELAY_INCORRECT: 10,
+		DATABASE_USER: __dirname + "/database/user.sqlite",
+		DATABASE_SLIMSTAMPEN: __dirname + "/database/slimstampen.sqlite",
+		ALGORITHM: "flashcard", // flashcard; slimstampen;
+		LANGUAGE: "nl",
+		ONLINE_HOST: "db4free.net",
+		ONLINE_USER: "enricdz148_slim",
+		ONLINE_DATABASE: "enricdz148_slim",
+		ONLINE_PASSWORD: "exVhqZJas"
```

Comment:
This isn't safe, but I don't know how to fixt it, as soon as I do I'll yet know.